cli/exp_scaletest.go

```diff
@@ -957,13 +966,34 @@ func (r *RootCmd) scaletestWorkspaceTraffic() *clibase.Cmd {
 					}
 					agentID = res.Agents[0].ID
 					agentName = res.Agents[0].Name
+					apps = res.Agents[0].Apps
 				}
 
 				if agentID == uuid.Nil {
 					_, _ = fmt.Fprintf(inv.Stderr, "WARN: skipping workspace %s: no agent\n", ws.Name)
 					continue
 				}
 
+				if app != "" {
```

Comment:
nit: maybe consider slicing this block into functions for readability purposes. For example: `configureWorkspaceApp()`